### ISSUE TITLE
Event details UI not as expected for event series

### ISSUE THREAD

**Issue by saumier** [culturecreates/minimal-footlight-search-widget#76]
This Mariana Mazza event series does not have a good user experience.  Very hard to see the next performance date.  

@troughc Please take a look and take action with Minimal if you think this should be addressed.

https://toutculture.stagingminimalmtl.com/event-detail/?event=64515af7e19f7200646d0c7a

http://demo.tout-culture.s3-website.ca-central-1.amazonaws.com/events/event-details.html?eventId=64515af7e19f7200646d0c7a

**Comment by troughc** [comment]
@saumier yes, i agree. Did you also see that there is problem with the times? The end time is wrong on the tout culture website (it should be 10PM, not 4PM). This was noted for other events in https://github.com/culturecreates/tech-support-tickets/issues/35

We should address this issue with the time and can perhaps bring up design....

cc @sahalali
